repository/src/main/resources/alfresco/subsystems/Search/noindex/noindex-search-context.xml

```diff
@@ -99,7 +99,13 @@
             <ref bean="search.indexerAndSearcherFactory" />
         </property>
     </bean>
-    
+
+    <bean id="search.fts.alfresco.switching" parent="base.search.fts.alfresco.switching" >
+        <property name="queryConsistency" value="${solr.query.fts.queryConsistency}"/>
+        <property name="hybridEnabled" value="${solr.query.hybrid.enabled}"/>

```

Comment:
I think it would make sense to introduce new generic properties for use as well as these:
```
... value="${search.query.fts.queryConsistency:${solr.query.fts.queryConsistency}}"...
```

Reply:
Thanks - makes sense (as per our previous walk-through). 

Now applied both here and in https://github.com/Alfresco/alfresco-enterprise-repo/pull/800

Reply:
See https://github.com/Alfresco/alfresco-enterprise-repo/pull/800#discussion_r873454822